README.md

```diff
@@ -57,6 +57,8 @@ There are (6) classifications of "things" included in this library:
 
 * Helper Functions (`crocks/helpers`): All other support functions that are either convenient versions of combinators or not even combinators at all cover this group.
 
+* Logic Functions (`crocks/logic`): A helpful collection of Logic based combinators. All of these functions work with predicate functions and let you combine them in some very interesting ways.
```

Comment:
A `Pred` is an actual ADT included in the lib. A predicate function is anything that takes a value and returns a `Boolean`. So I do not want to give the impression that they only work with the predicates `crocks` ships with. They will work for all predicates, even user defined ones. 
What should I link to to make it easier to understand?